docs/api/Makefile

```
@@ -43,9 +42,9 @@ html: Makefile migrate
 # Also, on macOS, need -i '' but not for Linux
 
 ifdef IS_MAC
-	find $(SOURCECOPYDIR)/modules/ -type f -name "*.md" -exec sed -i '' -e 's?\.md#?\.html#?g' {} \;
+	find $(SOURCECOPYDIR)api/modules/ -type f -name "*.md" -exec sed -i '' -e 's?\.md#?\.html#?g' {} \;
```

Given that it used to say `/modules`, shouldn't we prepend a slash to `api/modules` as well? (Same question for the `else` leg.)

```suggestion
	find $(SOURCECOPYDIR)/api/modules/ -type f -name "*.md" -exec sed -i '' -e 's?\.md#?\.html#?g' {} \;
```

---

Oh - SOURCECOPYDIR already ends with a / and on a mac, the extra slash of build/source//api  causes it to error.

Of course, I now see that I'm only changing in the api modules/ files and if files in the classes/ directory link to other files, this sed won't change it.

---

But - adding a "." to the "find  . $(SOURCECOPYDIR)"  should allows us to build with the extra "/" .  Am making the change now.

---

Oh - actually - just realized I clobbered logic I shouldn't have during the copy phase.